CMakeLists.txt

```diff
@@ -11,6 +11,10 @@ set(CMAKE_MODULE_PATH
 
 include(cmake/utils.cmake)
 
+# Set CFLAGS/CXXFLAGS
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -Wextra -Werror -std=gnu11")
```

Comment:
Is not gnu99 enough?

Reply:
Yes, you are right. I tried to use "c99" and it failed. gnu99 works fine. Fixed. Thanks!